iree/turbine/kernel/wave/utils.py

```diff
@@ -1082,10 +1094,18 @@ def to_default_device(tensor: torch.Tensor) -> torch.Tensor:
     return tensor.to(get_default_device())
 
 
+def device_arange(*args, **kwargs):
+    return to_default_device(torch.arange(*args, **kwargs))
+
+
 def device_randn(*args, **kwargs):
     return to_default_device(torch.randn(*args, **kwargs))
 
 
+def device_randn_like(*args, **kwargs):
```

Comment:
You don't need wrappers to `*_like` functions as they already inherit device.

Reply:
thanks!